### ISSUE TITLE
Can't click-and-open url links in zellij

### ISSUE THREAD

**Issue by LYK-love** [zellij-org/zellij#2425]
**Basic information**

`zellij --version`:  0.31.4
`uname -av` or `ver`(Windows): Darwin
``

List of programs you interact with:
`alacritty --version`: alacritty 0.10.1
iTerm2 version:  3.5.0

**Further information**
If I execute some local searver in my terminal, it wil usually give me a url, like `localhost:4000`, by clicking it, I can open that url in my explorer. This behavior is commen in explorers and terminals.
However, if such url shows in Zellij, no matter in "pane" or "tab", the url can't be opened by easily clicking it. It's presented just as normal text.
I have tested it in Zellij and tmux. In tmux, the "click-and-open url" behavior is normal, no matter in "window" or "pane". 
I also tested it in different terminal emulaters like Alacritty and iTerms2. The result has nothing to do with them.
So this problem must lie in Zellij itself, not related to other factors.

Here're some screenshots:

In tmux, this link can be clicked-and-opened:
<img width="614" alt="image" src="https://user-images.githubusercontent.com/74082944/236298521-839a04eb-b251-4b07-86cb-d892d96e79ed.png">

And in Zellij, when my mouse move to that link, I can't see any rendering which implies that I can click  and open it like in explorer.
<img width="708" alt="image" src="https://user-images.githubusercontent.com/74082944/236298883-4eedc63f-d423-48f9-a5e6-dddf59568aff.png">

(Due to the limit of my screenshot tool, my cursor image can't be recorded. But you can test it in your way.)

**Comment by jaeheonji** [comment]
I'm not the main user of `Alacritty`, so I don't know the details, but in `Windows Terminal`, URL links work even inside zellij. So now it seems to depend on Terminal's own feature.

So I consider this to be an enhancement rather than a bug.
Thanks for reporting. and I'll change the label.

**Comment by tlinford** [comment]
In tmux as far as I can tell, clicking without shift only works when mouse mode is off. If you turn it off in zellij it's the same.

**Comment by d1onys1us** [comment]
> In tmux as far as I can tell, clicking without shift only works when mouse mode is off. If you turn it off in zellij it's the same.

yeah i had the same behavior here. however if i am using multiple vertical panes the link is "cut off" (like if the link gets word wrapped, it only copies the part up until the border of the pane) and when i try to click it, it only opens up the cut off version of url. i'm wondering if there is a workaround for this?

**Comment by tlinford** [comment]
ah i see.. that's a bug!

**Comment by alissonbrunosa** [comment]
> In tmux as far as I can tell, clicking without shift only works when mouse mode is off. If you turn it off in zellij it's the same.

Hey @tlinford,
You're right, I completely missed that mouse support is off by default on TMUX. 
Can confirm that the behaviour is the same with `mouse_mode false` on Zellij

**Comment by Vatson112** [comment]
Hi @alissonbrunosa 

I confirm this behavior  again with

```bash
zellij 0.39.2
alacritty 0.13.1
```

With `mouse_mode true`  links are not opened by clicking it.

With `mouse_mode false` links opened normally.

It would be nice to open links in both modes.

**Comment by yarcod** [comment]
> ... i am using multiple vertical panes the link is "cut off" (like if the link gets word wrapped, it only copies the part up until the border of the pane) and when i try to click it, it only opens up the cut off version of url. i'm wondering if there is a workaround for this?

I just wanted to chime in that I've experienced this exact thing as well. Clicking links in general do work for me however, using latest Zellij and Alacritty.

**Comment by HaroldArpanet** [comment]
Hey
I have the same issue in Alacritty.
Versions:
- alacritty 0.13.2
- zellij 0.40.1

**Comment by tunix** [comment]
I have this issue with ghostty (1.1.0) + zellij (0.41.2).

**Comment by archcorsair** [comment]
I'm using Ghostty on macOS:
As long `mouse_mode: true` is on you can open a link with `CMD + Shift + Click`

**Comment by sapeurfaire** [comment]
I'm using Ghostty on Linux.
If mouse_mode is enabled, must `CTRL + Shift + Click` to open a URL.
If mouse_mode is disabled, just `Shift + Click` will open a URL.
Mirroring sentiment of @Vatson112  would be nice if the behavior were consistent.  Cheers!

**Comment by bertoabist** [comment]
What about opening a file path in default editor? It's not working for me. 
I'm using Ghostty on macOs
